src/functions.php

```diff
@@ -0,0 +1,54 @@
+<?php
+require_once 'src/db_structures.php';
+//function that converts the dogs array into its individual fields from the db_functions
+function returnDogs(array $dog_data): string
+{
+    $dog_string = '';
+
+    foreach ($dog_data as $dog) {
+        if ($dog['owner_id'] == 2) {
+            $belongs = 'I belong to Finn';
+
+        } else {
+            $belongs = 'I belong to Nan';
+        }
+        $dog_string .=
+            '<div class="collection_item">' .
+            '<h4>Dog name:</h4>' . '<p>'.$dog['name'].'</p>' .
+            '<h4>Dog breed:</h4>' . '<p>'.$dog['breed'].'</p>' .
+            '<em>'.$belongs.'</em>' .
+            '</div>  
+            <img class="collection_item_image" src="' . $dog['image'] . '">';
+    } return $dog_string;
```

Comment:
```suggestion
    } 
    return $dog_string;
```

Curly brace finished the foreach, returning is another thing - more readable if these different things are on their own lines.